svelte-cloudinary/src/lib/components/CldImageTypes.ts

```diff
@@ -0,0 +1,12 @@
+// This types still comes wrong
+// import type { ImageProps } from '@unpic/svelte'
+// But this is the same used by the @unpic/svelte package
+import type { UnpicImageProps } from '@unpic/core';
+import type { HTMLImgAttributes } from 'svelte/elements';
+type ImageProps = UnpicImageProps<HTMLImgAttributes, string | null>;
+import type { ImageOptions } from '@cloudinary-util/url-loader';
+export type CldImageProps = ImageOptions &
+  ImageProps & {
+    layout?: ImageProps['layout'];
+    tint?: string;
```

Comment:
@matiasfha is `tint` needed here?

Reply:
Yes, tint is not exposed by ImageOptions type but it can be used as a prop of the component, by adding the optional tint prop there the user of CldImage can see it without TS error on it

Reply:
got it. it's a bug that it's not on ImageOptions, i opened an Issue to track that, but it needs to be applied via the URL Loader package

https://github.com/cloudinary-community/svelte-cloudinary/issues/29